client/src/main/com/sinch/sdk/domains/verification/models/requests/StartVerificationCalloutRequestParameters.java

```diff
@@ -0,0 +1,64 @@
+package com.sinch.sdk.domains.verification.models.requests;
+
+import com.sinch.sdk.core.models.OptionalValue;
+import com.sinch.sdk.domains.verification.models.Identity;
+import com.sinch.sdk.domains.verification.models.VerificationMethodType;
+import com.sinch.sdk.domains.verification.models.VerificationReference;
+
+/**
+ * Dedicated request parameters to be used for a CALLOUT verification
+ *
+ * @since 1.0
+ */
+public class StartVerificationCalloutRequestParameters extends StartVerificationRequestParameters {
+
+  private StartVerificationCalloutRequestParameters(
+      OptionalValue<Identity> identity,
+      OptionalValue<VerificationReference> reference,
+      OptionalValue<String> custom) {
+    super(identity, OptionalValue.of(VerificationMethodType.CALLOUT), reference, custom);
```

Comment:
Do you need to keep the `method` parameter as an Optional?

Reply:
Really not.
Thank you
Fixed